### PR TITLE
[SRVKS-1220] Readiness and liveness probing on a multi-container service

### PR DESCRIPTION
**Pull request by ochromy**
Version(s):
 serverless-docs-1.32+

Issue:
https://issues.redhat.com/browse/SRVKS-1220

Link to docs preview:
https://76863--ocpdocs-pr.netlify.app/openshift-serverless/latest/knative-serving/config-applications/multi-container-support-for-serving.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mgencur** [comment]
LGTM from QE

File: modules/serverless-probing-multi-container-service.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+...
+spec:
+  config:
+    features:
+      "multi-container-probing": enabled <1>
+...
+----
+<1> Enabled multi-container-probing feature 
+
+. Apply the updated `KnativeServing` CR
```
Comment (kcarmichael08) [inline modules/serverless-probing-multi-container-service.adoc:31]:
```suggestion
. Apply the updated `KnativeServing` CR.
```

File: modules/serverless-probing-multi-container-service.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+         readinessProbe: <1>
+           httpGet:
+             port: 8080
+       - name: second-container
+         image: gcr.io/knative-samples/helloworld-java
+         readinessProbe: <2>
+           httpGet:
+             port: 8090
+----
+<1> Readiness probe of the first container.
```
Comment (kcarmichael08) [inline modules/serverless-probing-multi-container-service.adoc:62]: I would omit periods for 1) and 2) because these aren't complete sentences

**Review by kcarmichael08 — COMMENTED** [review]
Hi, I marked a couple of style issues- lgtm otherwise!

**Review by kcarmichael08 — COMMENTED** [review]
Hi, I marked a couple of style issues- lgtm otherwise!

**Pull request merged**